index.html

```diff
@@ -284,6 +286,38 @@ <h2>Контакты</h2>
 
     <p class="htmlacademy-copyright">Разработано —<br><a href="http://htmlacademy.ru">htmlacademy.ru</a></p>
   </footer>
+  <section class="modal-write-us">
+    <h2 class="visually-hidden">Напишите нам!</h2>
+    <form action="https://echo.htmlacademy.ru" method="get">
+      <ul>
+        <li class="modal-write-us-name">
+          <p>Ваше имя:
+            <input type="text" name="name" id="name-place" placeholder="Имя Фамилия">
+            <label class="visually-hidden" for="name-place">Имя и фамилия</label></p>
+        </li>
+        <li class="modal-write-us-email">
+          <p>Ваше e-mail:
+            <input type="text" name="email" id="email-place" placeholder="email@example.com">
+            <label class="visually-hidden" for="email-place">Ваш e-mail</label></p>
+        </li>
+        <li class="modal-write-us-text">
+          <p>Текст письма:
+            <input type="text" name="written-text" id="written-text-place" placeholder="В свободной форме">
+            <label class="visually-hidden" for="written-text-place">Текст письма</label></p>
+        </li>
+      </ul>
+      <button class="modal-write-us-send" type="submit">Отправить</button>
+    </form>
+    <p><button class="modal-write-us-close" type="button">
+        <img src="img/svg/modal-cart-2.svg" alt="Закрыть форму обратной связи"></button> </p>
+  </section>
+  <section class="modal-popup-map">
+    <h2 class="visually-hidden">Карта с указанием адреса магазина - г. Санкт-Петербург, улица Большая Конюшенная, дом 19</h2>
```

Comment:
```suggestion
    <h2 class="visually-hidden">Карта с указанием адреса магазина: г. Санкт-Петербург, ул. Большая Конюшенная, д. 19</h2>
```
и ниже, по правилам русского языка пишется двоеточие.